### PR TITLE
py-utils: new package at 1.0.1

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-utils/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class PyUtils(PythonPackage):
+    """Useful, oft-repeated things for Python"""
+
+    homepage = "https://github.com/haaksmash/pyutils"
+    url      = "https://github.com/haaksmash/pyutils/archive/1.0.1.tar.gz"
+
+    version('1.0.1', sha256='3a6552db1678e86a1b5e31107d20ae90dc5fb858ff7936b95367ee9d1b99e9ae')
+    version('0.8.0', sha256='916672df4cf9647e44f0aa1b3b327eb361c3c0bac1b1e32a6cf723ca766a2d4b')
+
+    depends_on('python@3.6:', type=('build', 'run'), when='@0.9.0:')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-utils/package.py:18]:
```suggestion
    depends_on('python@3.6:', type=('build', 'run'), when='@1.0.0:')
```

File: var/spack/repos/builtin/packages/py-utils/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    depends_on('py-setuptools', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-utils/package.py:19]:
```suggestion
    depends_on('py-setuptools', type='build')
```

Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-utils/package.py]: updated

File: var/spack/repos/builtin/packages/py-utils/package.py (hunk 1 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-utils/package.py]: fixed!

**Pull request merged**